docs/dg/dev/third-party-module-development/develop.md

```diff
@@ -0,0 +1,159 @@
+{% endinfo_block %}
+
+**c.** Create a plugin that will provide data for the **Categories** column
+
+
+File Path: vendor/your-company-name/product-category/src/YourCompanyName/Zed/ProductCategory/Communication/Plugin/ProductManagement/ProductCategoryProductTableDataBulkExpanderPlugin.php
+
+```php
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace YourCompanyName\Zed\ProductCategory\Communication\Plugin\ProductManagement;
+
+use Generated\Shared\Transfer\ProductCategoryConditionsTransfer;
+use Generated\Shared\Transfer\ProductCategoryCriteriaTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\ProductCategory\Business\ProductCategoryFacade;
+use Spryker\Zed\ProductManagementExtension\Dependency\Plugin\ProductTableDataBulkExpanderPluginInterface;
+
+class ProductCategoryProductTableDataBulkExpanderPlugin extends AbstractPlugin implements ProductTableDataBulkExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Expands product table items with abstract product approval status.
+     *
+     * @api
+     *
+     * @param array<array<string, mixed>> $items
+     * @param array<array<string, mixed>> $productData
+     *
+     * @return array<array<string, mixed>>
+     */
+    public function expandTableData(array $items, array $productData): array
+    {
+        // The code below is code sample that just works. (Learn how to make it fancy in https://docs.spryker.com/docs/dg/dev/architecture/architectural-convention.html)
+        $productAbstractIds = array_map(function ($item){
+            return $item['id_product_abstract'];
+        }, $items);
+
+        $productCategoryCollection = (new ProductCategoryFacade())
+            ->getProductCategoryCollection(
+                (new ProductCategoryCriteriaTransfer())
+                    ->setProductCategoryConditions(
+                        (new ProductCategoryConditionsTransfer())
+                            ->setProductAbstractIds($productAbstractIds)
+                    )
+            );
+
+        foreach ($items as $key => $item) {
+            $productCategoryNames = [];
+
+            foreach ($productCategoryCollection->getProductCategories() as $productCategory) {
+                if ($productCategory->getFkProductAbstract() !== $item['id_product_abstract']) {
+                    continue;
+                }
+
+                $productCategoryNames[] = $productCategory->getCategory()->getName();
+            }
+
+            $items[$key]['categories'] = implode(', ', $productCategoryNames);
+        }
+
+        return $items;
+    }
+}
+
+```
+
+{% info_block infoBox %}
+Validate that on http://backoffice.de.spryker.local/product-management page you can see categories column with assigned categories.
+{% endinfo_block %}
+
+**Congratulations! You've created your first Spryker module.**
```

Comment:
Mb we should also mention that the new module should be added in composer.json on project level?

Reply:
At this point we don't even have github repository yet, does it make sense?
I would suggest to add it it `publish-code.md` file, what do you think?